runExperimentAbundanceCQF.sh

```diff
@@ -0,0 +1,50 @@
+readsFile=$1
+percent=$2
+sketchSize=$3
+outputPrefix=$4
+
+
+countMinSketchSize=$(echo 100| awk '{print $1/4}')
+
+
+sketchFile=$(mktemp mmapExperiment.XXX.kh)
+sketchCQFFile=$(mktemp mmapExperiment.XXX.kh)
+
+distFile=$(mktemp /tmp/mmapExperiment.XXX.dist)
+distFileMMap=$(mktemp /tmp/mmapExperiment.XXX.dist)
+
+
+noTotalLines=$(wc -l $readsFile|cut -f1 -d ' ' )
+subset=$(echo $noTotalLines $percent | awk '{print ($1/4) * $2 * 4}')
+
+
+
+subsetFile=$readsFile
+
+head -n $subset < $readsFile > $subsetFile 
+
+echo "Counting Normal"
+
+python3 scripts/load-into-counting.py   -x $sketchSize $sketchCQFFile $subsetFile
+
+
+echo "Counting CQF"
```

Comment:
The file name, the echo messages, and the test folder are still referring to CQF instead of MQF.